src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformDispatcher.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.maui;
+
+import android.os.Handler;
+import android.os.Looper;
+
+/**
+ * Java-side support for Dispatcher.Android.cs
+ */
+public class PlatformDispatcher extends Handler {
+    private PlatformDispatcher(Looper looper) {
+        super(looper);
+    }
+
+    public static PlatformDispatcher create() {
+        Looper looper = Looper.myLooper();
+        if (looper == null || looper != Looper.getMainLooper())
+            return null;
+        return new PlatformDispatcher(looper);
+    }
+
+    public boolean isDispatchRequired() {
+        return Looper.getMainLooper() != getLooper();
```

Comment:
Comparing the old code, this should be:
```suggestion
        return Looper.myLooper() != getLooper();
```